trello2moment

```diff
@@ -148,34 +213,19 @@ async function moment (key, token, moment_name, board_name, output_file, images,
 
   let cards = board.cards;
 
-  if (images) {
-    let _imgName = output_file.split('/');
-    let imgName  = _imgName[_imgName.length - 1];
-    let encType  = imgName.split('.')[1];
-    let n = namedNode('');
-    if ( encType === 'ttl' ) {
-      add(n,rdf.type,schema.BasicContainer);
-    } else {
-      add(n,schema.publisher,namedNode('http://id.loc.gov/authorities/names/no2008108707'));
-      add(n,schema.license,namedNode('http://rightsstatements.org/vocab/CNE/1.0/'));            
-      add(n,schema.encodingFormat,literal('image/' + encType));
-      add(n,schema.name,literal(imgName));
-    }
-  } else {
-    set_card_type_from_lists(board.lists);
-    for(let k=0;k<cards.length;k++) {
-      await add_card(cards[k]);
-    }
+  if ( thumbnails ) {
+    getImages(moment_name,cards);
+  }
+
+  set_card_type_from_lists(board.lists);
+  for(let k=0;k<cards.length;k++) {
+    await add_card(cards[k]);
   }
 
   writer.end((error, result) => {
-    if (program.output_file) {
-      fs.writeFile(program.output_file, result, (err) => {
-        if (err) throw err;
-      });
-    } else {
-      console.log(result);
-    }
+    fs.writeFile(program.moment+'_moment.ttl', result, (err) => {
```

Comment:
Removed --output-file, and always write to the same place

Reply:
Got it! :)